### DISCUSSION TITLE
How can I keep image tags/likes across file moves?

### DISCUSSION THREAD

**Discussion by FelixDombek** [zanllp/infinite-image-browsing#754] [Q&A] [answered]
Is there any mechanism that allows me to retain image information such as likes when I rename the `outputs/txt2img/...` subfolder?

I guess I could write a script to open the SQLite db and change the paths there while the app is not running, but I hope there's a simpler way. Thanks for your help.

**Comment by zanllp** [comment, marked as answer]
https://github.com/zanllp/sd-webui-infinite-image-browsing/wiki/Proper-Folder-Migration-to-Avoid-Data-Loss---%E6%AD%A3%E7%A1%AE%E7%9A%84%E8%BF%81%E7%A7%BB%E6%96%87%E4%BB%B6%E5%A4%B9%E9%81%BF%E5%85%8D%E6%95%B0%E6%8D%AE%E4%B8%A2%E5%A4%B1

**Comment by FelixDombek** [comment]
Thanks! This is exactly what I need. 
> you can directly use the drag-and-drop or move functionality in IIB

How do I access that? When I drag a folder or image, I just see "add to comparison".

**Comment by zanllp** [comment]
https://github.com/zanllp/sd-webui-infinite-image-browsing?tab=readme-ov-file#transfer-files-between-different-tab-panes

![image](https://github.com/user-attachments/assets/d0c4e0f3-4a6e-4075-953f-20370ffcbbb2)

**Comment by zanllp** [comment]
Or via right-click menu  
<img width="253" alt="image" src="https://github.com/user-attachments/assets/a9ba639b-0ed4-46c5-912d-2b0973b4d591">

**Comment by FelixDombek** [comment]
Thanks, I got it working with the two side by side panes! 

If anyone has the same question: In the "new tab" view, I right clicked on a folder under _"Normal / Fixed Mode"_, selected _"Open to the side"_, then navigated into the folder where the images should end up. Then select the images in the left pane and drag-drop them into the right pane. (Dragging them onto a folder icon does apparently not place them into that folder. You need to enter the folder first.)

The _Copy To/Move To_ menu doesn't show subfolders so that didn't work for me. I realize now I could manually add the subfolder to the _"Normal / Fixed mode"_ panel, but that would get tedious for many moves across many different subfolders. Maybe adding ">" symbols to further unfold submenus would be a nice addition.
